### PR TITLE
Custom API keys with Custom Admin Forms

### PR DESCRIPTION
**Pull request by mabe13**

### CODE REVIEW

File: application/api/capacityauth/admin.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# organizations/admin.py
+import typing
+from django.forms import ModelForm
+from django.contrib import admin
+from django.db import models
+from django.http.request import HttpRequest
+from rest_framework_api_key.admin import APIKeyModelAdmin
+from .models import DosUserAPIKey
+
+class DosUserAPIKeyAdminForm(ModelForm):
```
Comment (ntoll) [inline application/api/capacityauth/admin.py]: So the ModelForm needs a Model instance to work properly, right..? See: https://docs.djangoproject.com/en/3.0/topics/forms/modelforms/ This is usually done via the `Meta` class into which you can say the model that's being used as the basis for the form and the fields which you want to use from the model within the form you're specifying. E.g.:

```
class Meta:
    model = DosUserAPIKey
    fields = ["foo", "bar", "baz", ]  # Only these fields will be displayed in the form.
```

The docs I link to above tell you how this all works.

File: application/api/capacityauth/admin.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+class DosUserAPIKeyModelAdmin(APIKeyModelAdmin):
+
+    form = DosUserAPIKeyAdminForm
+    is_change_view_call = False
+
+    def get_exclude(self, request, obj=None):
+        if self.exclude == None:
+            self.exclude = ["dosuserid", "name"]
+        else :
+            self.exclude = self.exclude + ["dosuserid", "name"]
```
Comment (ntoll) [inline application/api/capacityauth/admin.py]: Why not..?

```
if self.exclude:
    self.exclude += ["dosuserid", "name"]  # or perhaps more explicit: self.exclude.extend(["dosuserid", "name", ])
else:
    self.exclude = ["dosuserid", "name"]
```

(FWIW, `None` is to False-y and a list with items in it is Truth-y)

Comment (mabe13) [reply application/api/capacityauth/admin.py]: Good spot 👍  changed in latest commit

**Comment by mabe13** [comment]
Co-coded and co-review by me @mabe13 and @jope1, therefore is ready to be merged into master.

**Pull request merged**